sechub-pds-solutions/gitleaks/tests/copy_sanitycheck_files.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+current_test_folder="$1"
+	
+if [[ ! -d "$current_test_folder" ]]
+then
+	echo "Target folder is empty"
+	exit 1
+fi
+
+echo "copy sanity check testdata folder"
+cp "./sanity-check-testdata" "$current_test_folder/sanity-check" -r
```

Comment:
The `./` can be omitted here; please place the flags before the arguments
```suggestion
cp -r sanity-check-testdata/ "$current_test_folder/sanity-check"
```